node/runtime/src/constants.rs

```diff
@@ -20,7 +20,10 @@
 pub mod currency {
 	use node_primitives::Balance;
 
-	pub const MILLICENTS: Balance = 1_000_000_000;
+	pub const NANO: Balance = 1;
+	pub const MICRO: Balance = 1_000 * NANO;
+	pub const MILLI: Balance = 1_000 * MICRO;
+	pub const MILLICENTS: Balance = 1_000 * MILLI;
```

Comment:
Clarification:

1 Nano = 10^-9 COIN(RING/KTON), 1 Dollar = 1 COIN(RING/KTON).

So we may not user terms ```MILLICENTS, CENTS, DOLLARS ``` anymore.

Reply:
Ok, I'll update the name.